assembly/JSON.ts

```diff
@@ -246,6 +306,21 @@ export namespace JSON {
       this.keys = new Array();
     }
 
+    toString(): string {
+      const objs: string[] = [];
+      for (let i: i32 = 0; i < this.keys.length; i++) {
+        objs.push(
+          '"' + this.keys[i] + '":' + this._obj.get(this.keys[i]).toString()
```

Comment:
Do you have any tests for this?  Given the error above in value's `toString` it seems like since the value isn't cast to the subclass it should fail.